buildenv/jenkins/testJobTemplate

```diff
@@ -326,8 +326,16 @@ ARCH_OS_LIST.each { ARCH_OS ->
 							    sectionHeaderStyle(sectionHeaderHelpTextStyleCss)
 							}
 							choiceParam('TEST_FLAG', ['', 'JITAAS', 'AOT'], "Optional. Only set to JITAAS/AOT for JITAAS/AOT feature testing.")
-							stringParam('EXTRA_OPTIONS', EXTRA_OPTIONS, "Use this to append options to the test command")
-							stringParam('JVM_OPTIONS', "", "Use this to replace the test original command line options")
+							stringParam('EXTRA_OPTIONS', EXTRA_OPTIONS, "Use this to append options to the commandline",'''
+							Example 1 :<br>
+							Extra_Options='-Xjit:count=0,\{java/lang/reflect/Method.getParameterAnnotations*\}\(traceFull,log=getParameterAnnotations.log\)'
+							Example 2:<br>
+							-Xjit:\"{java/util/HashMap.*}(optlevel=noopt)\"
+							''')
```

Comment:
stringParam takes three parameters:
`stringParam(String parameterName, String defaultValue = null, String description = null)`

It is incorrect to add an additional parameter. The example should be a part of the description.
Also, the examples are specific to system tests, we should clearly state this.


```
Use this to append options to the commandline. For system tests, the special characters may need to be escaped in EXTRA_OPTIONS and JVM_OPTIONS. 
Example 1:
'-Xjit:count=0,\{java/lang/reflect/Method.getParameterAnnotations*\}\(traceFull,log=getParameterAnnotations.log\)'
Example 2:
-Xjit:\"{java/util/HashMap.*}(optlevel=noopt)\"
```

We can use the same example for  `JVM_OPTIONS`.

Reply:
Thank you @lxia. I didn't know this format. I have corrected the mistake. Please review. If there are any more mistakes I will correct them immediately.